debs/SPECS/3.10.0/wazuh-agent/debian/postrm

```diff
@@ -55,6 +49,12 @@ case "$1" in
     ;;
 
     purge)
+        if getent passwd | grep -q "^ossec" ; then
```

Comment:
This change is ok but the indentation in some lines are tabs and in other are spaces. Can you change them to spaces?